readme.md

```diff
@@ -1,151 +1,136 @@
-- [Bitbucket Server](https://docs.renovatebot.com/modules/platform/bitbucket-server/)
-- [Azure DevOps](https://docs.renovatebot.com/modules/platform/azure/)
-- [AWS CodeCommit](https://docs.renovatebot.com/modules/platform/codecommit/)
-- [Gitea and Forgejo](https://docs.renovatebot.com/modules/platform/gitea/)
-- [Gerrit (experimental)](https://docs.renovatebot.com/modules/platform/gerrit/)
+## Ways to run Renovate
 
-## Who Uses Renovate?
+The most effective way to run Renovate is to use an automated job scheduling system that regularly runs Renovate on all enabled repositories and responds with priority to user activity.
+Mend offers cloud-hosted and self-hosted solutions.
+See the options below.
 
-Renovate is widely used in the developer community:
+### Mend Renovate Community (Cloud-Hosted)
 
-![Logos of companies and projects that use Renovate](./docs/usage/assets/images/matrix.png)
+**Supports: GitHub.com, Bitbucket Cloud**
 
-<details>
-<summary>List of companies and projects that use Renovate</summary>
+Hosted by Mend.io.
+No setup is needed.
+Community plan available (Free)
 
-- Prisma
-- Netlify
-- Envoy
-- Condé Nast
-- Microsoft
-- Atlassian
-- Sourcegraph
-- Mozilla
-- Deloitte
-- Telus
-- Yarn
-- HashiCorp
-- Automattic
-- Algolia
-- eBay
-- Cypress
-- Red Hat
-- Financial Times
-- Uber
-- Buildkite
+- GitHub Cloud: Install the [Renovate Cloud-Hosted App](https://github.com/apps/renovate) on your GitHub org, then select the repos to enable
+- Bitbucket Cloud: Add the [Mend App](https://marketplace.atlassian.com/apps/1232072/mend) to your Workspace, then add the Mend Renovate user to the projects you want to enable
 
-</details>
+### Mend Renovate Community (Self-hosted)
 
-## Renovate OSS Insights
+**Supports: GitHub, GitLab, Bitbucket Data Center**
 
-Renovate is built on a big community and actively invites and supports contributions.
-Information about our contributors and community can be found on [OSS Insight](https://ossinsight.io/analyze/renovatebot/renovate).
+Install and run your own Renovate server.
+Access internal packages.
 
-## Star History
+- [Mend Renovate Community Self-Hosted](https://github.com/mend/renovate-ce-ee/tree/main/docs) (Free)
+- [Mend Renovate Enterprise](https://www.mend.io/mend-renovate/) (Paid plan)
 
-[![Star History Chart](https://api.star-history.com/svg?repos=renovatebot/renovate&type=Date)](https://star-history.com/#renovatebot/renovate&Date)
+### Other ways to run Renovate
 
-## The Renovate Approach
+If you can’t use a pre-built job scheduling system, or want to build your own, the following options are available:
 
-We believe everyone benefits from automation, whether it's a little or a lot.
-This means that Renovate:
+#### Run Renovate on your Pipeline
 
-- Adapts to your workflow
-- Allows you to configure its behavior
-- Will autodetect settings where possible
+Mend provides a _**GitHub Action**_ and _**GitLab Runner**_ to help you run Renovate as a CI pipeline job.
```

Comment:
```suggestion
Mend provides a _**GitHub Action**_ or a _**GitLab Runner**_ to help you run Renovate as a CI pipeline job.
```

This may be really stupid, but we want to avoid users thinking they need to use _both_ of these tools at once. :upside_down_face: 

So maybe replacing `and` with `or a` helps explain that you should use _one_ of these tools, not both?